system-tests/projects/npm-react-webpack/cypress.config.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  viewportWidth: 400,
+  viewportHeight: 400,
+  video: false,
+  experimentalFetchPolyfill: true,
+  component: {
+    excludeSpecPattern: [
+      '**/__snapshots__/*',
+      '**/__image_snapshots__/*',
+      'examples/**/*',
+      // due to how we handle system-tests, tests with hooks
```

Comment:
I'll try the `alias.resolve`, nice idea.

As for the `jsx` extension, we actually don't need it here since we've got 

```js
resolve: {
    extensions: ['.js', '.ts', '.jsx', '.tsx'],
```

In the `webpack.config.js`.